components/tools/OmeroPy/src/omero/plugins/chown.py

```diff
@@ -65,10 +75,65 @@ def cmd_type(self):
         import omero.all
         return omero.cmd.Chown2
 
+    def _configure(self, parser):
+        parser.set_defaults(func=self.main_method)
+        self._add_wait(parser, default=-1)
+        parser.add_argument(
+            "--include",
+            help="Modifies the given option by including a list of objects")
+        parser.add_argument(
+            "--exclude",
+            help="Modifies the given option by excluding a list of objects")
+        parser.add_argument(
+            "--ordered", action="store_true",
+            help=("Pass multiple objects to commands strictly in the order "
+                  "given, otherwise group into as few commands as possible."))
+        parser.add_argument(
+            "--list", action="store_true",
+            help="Print a list of all available graph specs")
+        parser.add_argument(
+            "--list-details", action="store_true", help=SUPPRESS)
+        parser.add_argument(
+            "--report", action="store_true",
+            help="Print more detailed report of each action")
+        parser.add_argument(
+            "--dry-run", action="store_true",
+            help=("Do a dry run of the command, providing a "
+                  "report of what would have been done"))
+        parser.add_argument(
+            "--force", action="store_true",
+            help=("Force an action that otherwise defaults to a dry run"))
+        self._pre_objects(parser)
+        parser.add_argument(
+            "obj", nargs="*", type=GraphArg(self.cmd_type()),
+            help="objects to be processed in the form <Class>:<Id>"
+                 " and/or user(s) to transfer all data from in the"
+                 " form Experimenter:<Id>")
+
     def _pre_objects(self, parser):
         parser.add_argument(
             "usr", nargs="?", type=ExperimenterArg,
-            help="""user to transfer ownership of objects to""")
+            help="user to transfer ownership of specified objects and/or all"
+                 " objects owned by specified user(s) to")
+
+    def populate_targetUsers(self, command_check):
+        # Move the Experimenters whose data are to be
+        # transferred from targetObjects
+        # to targetUsers. The rewritten _check_command
+        # method checked these Experiments when they were
+        # in targetObjects, but Chown2 needs them in targetUsers
+        for k, v in command_check.targetObjects.items():
```

Comment:
There *has* to be an easier way to do this in Python, something like,

```python
if "Experimenter" in command_check.targetObjects:
    command_check.targetUsers = command_check.targetObjects["Experimenter"]
    del command_check.targetObjects["Experimenter"]
```

or somesuch. (The above may not be it, but the correct thing will be correspondingly simple.)